source/MRMesh/MRPython.h

```diff
@@ -48,16 +49,6 @@ MR_ADD_PYTHON_CUSTOM_DEF( moduleName, name, [] (pybind11::module_& m)\
         def( "clear", &vecType::clear ); \
 } )
 
-#define MR_ADD_PYTHON_EXPECTED( moduleName, name, type, errorType )\
-MR_ADD_PYTHON_CUSTOM_DEF( moduleName, name, [] (pybind11::module_& m)\
-{\
-    using expectedType = tl::expected<type,errorType>;\
-    pybind11::class_<expectedType>(m, #name ).\
-        def( "has_value", &expectedType::has_value ).\
-        def( "value", ( type& ( expectedType::* )( )& )& expectedType::value, pybind11::return_value_policy::reference_internal ).\
-        def( "error", ( const errorType& ( expectedType::* )( )const& )& expectedType::error );\
-} )
```

Comment:
lets keep it for now (mb mark depricated or static assert inside)

Reply:
done by adding deprecation warning